### ISSUE TITLE
Infection Associated Leukopenia

### ISSUE THREAD

**Issue by irenamuffels** [obophenotype/human-phenotype-ontology#6793]
**Preferred term label:**
Infection Associated Leukopenia

**Synonyms**
Infection Associated Lymphopenia

**Definition (free text, please give PubMed ID)**
Decreased lymphocyte count during infections

**Parent term (use hpo.jax.org/app)**
Abnormal lymphocyte count

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:797
ORPHA:357237

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I am adding Infection Associated Lymphopenia as HP:4000034
I think that ``Infection Associated Leukopenia`` is a mistake and I am not adding it. If this is wrong, please reopen this request with more information.
